.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
`node_modules` is already in ignored by ESLint https://eslint.org/docs/user-guide/configuring/ignoring-code